UI/api-interface.cpp

```diff
@@ -620,6 +620,12 @@ struct OBSStudioAPI : obs_frontend_callbacks {
 		return bstrdup(recordOutputPath);
 	}
 
+	char *obs_frontend_get_translated_string(const char *string) override
+	{
+		QString out = QTStr(string);
```

Comment:
Why the conversion to QString? I think its better to return a const char* and use:
`return App()->GetString(string);`
or
`return Str(string);`

Reply:
Done